Tests/XCUITests/BaseTestCase.swift

```diff
@@ -69,6 +69,7 @@ class BaseTestCase: XCTestCase {
         } else {
             app.launchArguments = [LaunchArguments.PerformanceTest] + launchArguments
         }
+        app.launchArguments.append("-disableAnimations")
```

Comment:
We may want to leave the animations on for the performance tests.
https://github.com/mozilla-mobile/firefox-ios/pull/16342/files#r1322167633

Reply:
that's a good point. there's an issue with the tab performance tests where they are long running entirely because of an artifact of xcuitest, the background requirement for the app to be idle (i.e. the animations to be completed). But some sites like walmart.com have very long running animations, and it seems to be by design. some animations seem to be designed to intentionally run perpetually.

so we need to find a workaround that could be as simple as making sure the first tab isn't walmart.com.

Reply:
with that said, we can add a launch argument in the test setup just for the performance tests, i suppose, that enables animations. that way they're still disabled for everything else.

Reply:
In theory, with L40 using launchArguments this should be already disabled.

Reply:
Since we have a set up in the performance tests, we should include it as:
` LaunchArguments.DisableAnimations` after L28

Reply:
just added it!